setup.cfg

```diff
@@ -3,3 +3,9 @@ python-file-with-version = asv/__init__.py
 
 [tool:pytest]
 addopts=-p no:logging
+
+[flake8]
+exclude=extern/
+max-line-length = 80
```

Comment:
This could be bigger, maybe 120.

Reply:
It's not up to me, but the default is 79 and that's pretty standard.  I only bumped it to 80 here because there are a lot of lines in asv that are exactly 80.

Reply:
I recommend you change it to a larger number. 80 may be a limit, but there's no particular reasoning for that particular number. Indeed, asv codebase at several places has longer lines, which I don't think is problematic for readability.